Cynosura.Studio.CliTool/CliApp.cs

```diff
@@ -45,11 +49,16 @@ public CliApp(string[] args)
             var defaultConfig = new Dictionary<string, string>
             {
                 {"Nuget:FeedUrl", _feed ?? "https://api.nuget.org/v3/index.json"},
+                {"Nuget:Username", _feedUsername},
+                {"Nuget:Password", _feedPassword},
                 {"LocalFeed:SourcePath", _src}
             };
+            var useProfileSettings =
+                Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile), ".cynosura.json");
```

Comment:
Let's change it to Cynosura.Studio/appsettings.json

Reply:
what if `.cynosura/appsettings.json` in user profile directory?

Reply:
ok

Reply:
Done